File: Naloga 14/naloga_14.py
def naloga_13():
    besedilo = input("vpisi besedilo: ")
    dolzina = len(besedilo)
    if dolzina > 2:
        stevec = 2
        while stevec < dolzina:
            print(besedilo[stevec])
            stevec = stevec + 3

if __name__ == '__main__':
    naloga_13()

File: Naloga 14/naloga_14.cpp
#include <iostream>
void naloga_14() {
    std::string besedilo;
    std::cin >> besedilo;
    if (besedilo.length() > 2){
        for (int stevec = 2; stevec < besedilo.length();) {
            std::cout << besedilo[stevec] << std::endl;
            stevec = stevec+3;
        };
    };
};
int main() {
    naloga_14();
    return 0;
}